lib/standard_api/controller.rb

```diff
@@ -131,7 +137,7 @@ def current_mask
     end
 
     module ClassMethods
-    
+
```

Comment:
Why you commit whitespace, if you want do separate commit. When you go through your commit read again and just stage the actual changes